run.go

```diff
@@ -146,6 +147,9 @@ func run(context *cli.Context) error {
 		// close and restore console
 		console.Close()
 		term.RestoreTerminal(os.Stdin.Fd(), consoleState)
+
+		//runtime should forward container exit code to the system
+		os.Exit(ps.Sys().(syscall.WaitStatus).ExitStatus())
```

Comment:
It looks like cli actually provides an interface for this: https://github.com/urfave/cli#exit-code, so we should probably use that.